cpu/cc26xx/dev/cc26xx-rf.c

```diff
@@ -92,6 +92,12 @@
 #define PRINTF(...)
 #endif
 /*---------------------------------------------------------------------------*/
+#if !defined(CC26XX_DEBUG_CRC) && defined(DEBUG)
```

Comment:
How about something like this instead of !checks:

```
#ifdef CC26XX_CONF_DEBUG_CRC
#define CC26XX_DEBUG_CRC CC26XX_CONF_DEBUG_CRC
#else /* CC26XX_CONF_DEBUG_CRC */
#define CC26XX_DEBUG_CRC DEBUG
#endif /* CC26XX_CONF_DEBUG_CRC */
```